app/internal/page/certificate_provider_details.go

```diff
@@ -104,6 +108,9 @@ func (d *certificateProviderDetailsForm) Validate() map[string]string {
 	if _, ok := errors["date-of-birth"]; !ok && d.DateOfBirthError != nil {
 		errors["date-of-birth"] = "dateOfBirthMustBeReal"
 	}
+	if d.Mobile == "" {
```

Comment:
should we check it starts with 07 or +447 then has 9 digits following?

Reply:
I mean, it wasn't on the ticket so... 😂 yeah good point, I'll add in.